charms/worker/k8s/src/config/extra_args.py

```diff
@@ -58,37 +70,34 @@ def craft(
             The configuration object to be updated with extra arguments.
         cluster_name (str): the name of the cluster to override in the extra arguments.
         node_ips (list[str]): the IP address of the node to override in the extra arguments.
+        datastore(Optional[str]): the name of the datastore.
```

Comment:
its not in the snap form, it's in the charm form right?

```suggestion
        datastore(Optional[str]): the name of the datastore in charm notation
```